gcp/rakefiles/vars.rb

```diff
@@ -2,6 +2,9 @@
 require "yaml"
 
 class Vars
+
+  SENSITIVE = "<sensitive>"
```

Comment:
No need for this to be a class variable. Please move it to `set_secrets()`, the only method that uses it.